.circleci/config.yml

```diff
@@ -0,0 +1,19 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:6.11.1
+
+    steps:
+      - checkout
+      - restore_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
```

Comment:
The other thing we might want to do is add a `v1-` prefix to all of our cache keys.  That way, if we need to, we can bust the cache by bumping the version to `v2-` and committing the change.  We might want to do that when we upgrade node versions or something like that.